paperscraper/lib.py

```diff
@@ -78,7 +78,7 @@ def format_bibtex(bibtex, key, clean: bool = True) -> str:
         entry = style.format_entry(label="1", entry=bd.entries[key])
         return entry.text.render_as("text")
     except Exception:
-        return bd.entries[key].fields["title"]
```

Comment:
What if the title does exist though? Shouldn't we try to return that first.

Reply:
Yeah I hear what you're saying, I was thinking about this too. I guess it depends if one considers a title alone a valid citation.

I think if one goes with just title, callers don't have an easy way to know to re-attempt citation generation themselves.

What do you think? I am down with sticking with title as a primary fallback if you think it's better.

Reply:
Another observation is falling back on DOI doesn't work because DOI isn't present in some BibTeX either :S

Reply:
Okay, what I do is let the failure (no title) continue to propagate, but I made it a more meaningful `CitationConversionError`